configuration/config.yml

```
@@ -31,3 +31,8 @@ certificateExpiryDateCheckServiceConfiguration:
   enable: true
 certificateOcspRevocationStatusCheckServiceConfiguration:
   enable: true
+selfServiceConfiguration:
```

This should be `selfService`
```suggestion
selfService:
```